src/client/app/containers/BarChartContainer.js

```diff
@@ -74,6 +74,11 @@ function mapStateToProps(state) {
 			callbacks: {
 				label: tooltipItems => `${data.datasets[tooltipItems.datasetIndex].label}: ${tooltipItems.yLabel} kWh`
 			}
+		},
+		plugins: {
+			datalabels: {
+				display: false
```

Comment:
Can you set this to global state somewhere in chartjs instead of individual instances?